docs/tutorial_tripole_grid.ipynb

```diff
@@ -996,22 +996,215 @@
    "metadata": {},
```

Comment:
<p>As per one of Ryan's previous reviews, it would be better to use a context manager for file writing:</p><pre class="ql-syntax" spellcheck="false">with open('data.nc', 'wb') as fp:
    fp.write(file.content)    
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NoraLoose/gcm-filters/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>